### ISSUE TITLE
As a user, I cannot spam the customer support email inbox since my messages and complaints go through the Talkback channel.

### ISSUE THREAD

**Issue by pronom8** [OHTU-German-learning-website/OHTU-German-learning-website#231]
The user can send 1 message per hour, using the talkback channel form.
